Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM nginx:1.19
+RUN rm -rf /etc/nginx/conf.d/default.conf
+RUN rm -rf /docker-entrypoint.d/*
```

Comment:
Are you happy with the default settings in `/etc/nginx/nginx.conf`? I often take inspiration from https://github.com/h5bp/server-configs-nginx

Reply:
What do you mean? I copy my own file with settings a few lines below.

Reply:
No, you copy `/etc/nginx/conf.d/nginx.conf`. `/etc/nginx/nginx.conf` is still the original from the image.